app/uk/gov/hmrc/cgtpropertydisposalsfrontend/controllers/returns/representee/RepresenteeController.scala

```diff
@@ -205,6 +211,43 @@ class RepresenteeController @Inject() (
     }
   }
 
+  def enterDateOfDeath(): Action[AnyContent] = authenticatedActionWithSessionData.async { implicit request =>
+    withCapacitorOrPersonalRepresentativeAnswers(request) { (representativeType, journey, answers) =>
```

Comment:
we should make sure that this page is only displayed for personal representatives - we don't ask capacitors this question. You could look at `representativeType` here and redirect to cya if you find a `Capacitor`. Same thing for the submit endpoint